src/components/TokenSelect/CommonBases.tsx

```diff
@@ -0,0 +1,81 @@
+import { Currency } from '@uniswap/sdk-core'
+import Row from 'components/Row'
+import TokenImg from 'components/TokenImg'
+import { BASES_TO_CHECK_TRADES_AGAINST } from 'constants/routing'
+import styled, { css } from 'styled-components/macro'
+import { ThemedText } from 'theme'
+import { currencyId } from 'utils/currencyId'
+
+const BasesContainer = styled(Row)`
+  margin: 0 1.25em;
+`
+
+const activeCss = css`
+  background-color: ${({ theme }) => theme.activeSoft};
+  border-color: ${({ theme }) => theme.active};
+  color: ${({ theme }) => theme.active};
+`
+
+const BaseWrapper = styled(Row)<{ active?: boolean }>`
+  border: 1px solid ${({ theme }) => theme.outline};
+  border-radius: 1em;
+  color: ${({ theme, active }) => (active ? theme.active : theme.primary)};
+  cursor: pointer;
+  padding: 0.5em 0.75em 0.5em 0.5em;
+
+  ${({ active }) => active && activeCss};
+
+  :hover,
+  :focus {
```

Comment:
Should this be focus-within, like in inputCss? Test by tabbing.

Reply:
no, tabbing is working for me like this